src/Drupal/Commands/core/WatchdogCommands.php

```diff
@@ -275,14 +281,16 @@ public function showOne($id, $options = ['format' => 'yaml']): PropertyList
      *   String. Valid watchdog type.
      * @param $severity
      *   Int or String for a valid watchdog severity message.
+     * @param $severity_min
+     *   Int or String for the minimum severity to return.
      * @param $filter
      *   String. Value to filter watchdog messages by.
      * @param $criteria
      *   ('AND', 'OR'). Criteria for the WHERE snippet.
      * @return
      *   An array with structure ('where' => string, 'args' => array())
      */
-    protected function where($type = null, $severity = null, $filter = null, $criteria = 'AND'): array
+    protected function where($type = null, $severity = null, $severity_min = null, $filter = null, $criteria = 'AND'): array
```

Comment:
Lets move severity_min to the end just in case someone is calling this method from custom code.

Reply:
Yes, done.